xml/deployment_hwrecommend.xml

```diff
@@ -71,12 +79,20 @@
   <sect2 xml:id="ceph-install-ceph-deploy-network">
    <title>Network Recommendations</title>
    <para>
-    For the &ceph; network environment, we recommend two bonded 25 GbE (or
-    faster) network interfaces bonded using 802.3ad (LACP). The use of two
-    network interfaces provides aggregation and fault-tolerance. The bond
-    should then be used to provide two VLAN interfaces, one for the public
-    network, and the second for the cluster network. Details on bonding the
-    interfaces can be found in
+     We recommend a single fault-tolerant network with enough bandwidth to
+     fulfil your requirements. For the &ceph; public network environment, we
+     recommend two bonded 25&nbsp;GbE (or
+     faster) network interfaces bonded using 802.3ad (LACP). This is considered
+     the minimal setup for &ceph;. If you are also using a cluster network, we
+     recommend four bonded 25&nbsp;GbE network interfaces. Doubling the network interfaces
+     provides aggregation and fault-tolerance.
```

Comment:
This might be nit-picking but I stumbled somewhat over that last sentence. Maybe being a bit more explicit here can add value?
```
Bonding two or more network interfaces provides better throughput via link aggregation and, given redundant links and switches improved fault tolerance and maintainability.
```